discover/portal/articles/12-using-workflow/02-creating-new-workflow-definitions.markdown

```diff
@@ -501,7 +501,6 @@ were at their desk that hadn't reviewed content assigned to them.
 					]]>
 				</script>
 				<script-language></script-language>
-				<execution-type></execution-type>
```

Comment:
@natocesarrego for my own curiosity, is a timer-execution-type appropriate here? 

<xs:simpleType name="timer-execution-type">
		<xs:restriction base="xs:string">
			<xs:enumeration value="onTimer" />
		</xs:restriction>
	</xs:simpleType>

Reply:
Actually, It's unnecessary, because the Workflow code just uses the elements <duration> and <scale> to trigger a timer. This timer-execution-type only have one valid value, which is onTimer and it represents the moment the timer count finishes. When the timer count finishes, then the Workflow will execute the configured actions, notifications and/or reassignments. Although, apparently, this is not being used by the Workflow definitions.

Reply:
Oh okay, I think I see. Whether or not you specify a timer-execution-type of onTimer, the timer is always going to proceed to its actions once the duration is up anyway. So this is really unnecessary. Thanks!

Reply:
Exactly!